components/cards/Work.jsx

```diff
@@ -6,17 +6,17 @@ import { DefaultButton } from "../ui/Button";
 const Work = ({ text, title, imageSrc2 }) => {
   return (
     <div className="pt-6 pb-8 xl:pt-32.75 xl:pb-27.5 bg-primary-800 w-full xl:w-320 rounded-xl flex-col lg:flex lg:flex-row md:justify-between px-4 xl:px-0 xl:pl-12.75 ">
```

Comment:
consider removing white space after `xl:pl-12.75` class

Reply:
thanks for pointing out, it's resolved!!